SparkPost/2web2ui
# FE-703 - Color and speed-up our test-ci script

[FE-703](https://jira.int.messagesystems.com/browse/FE-703)

### What Changed
- Added `--colors` flag to Jest command
- Updated Jest to latest version
- Updated Node to more recent version in the CI config
- The `--runInBand` flag produced no discernible speed improvements
- The `--maxWorkers` flag can't be used with our current Travis plan - with a paid plan we could potentially use it

### How To Test
- Check out the Travis output to see colors: https://travis-ci.org/SparkPost/2web2ui/builds/575330979?utm_source=github_status&utm_medium=notification
- Compare the test runtime between two Travis builds:
    * FE-703: https://travis-ci.org/SparkPost/2web2ui/builds/577855657
    * Master: https://travis-ci.org/SparkPost/2web2ui/builds/577567808

### To Do
